release-notes/unreleased/IGAPP-965-inappbrowser-already-opened-error.yml

```
@@ -0,0 +1,6 @@
+issue_key: IGAPP-965
+show_in_stores: true
+platforms:
+  - ios
+en: Fix multiple inappbrowser opened.
+de: Fehler mit mehreren gleichzeitig geöffneten InAppBrowsern behoben.
```

I think this error was not visible to users, was it? If so I'd change it to 
```suggestion
show_in_stores: false
platforms:
  - ios
en: Fix multiple inappbrowser opened.
```
and add a `not-testable` label to the JIRA issue.